designdocs/capacity.md

```diff
@@ -434,31 +468,38 @@ pub struct CapacityDetails<Balance> {
 
 ```
 
-### **Traits**
+#### **Traits**
 
 As mentioned above, Capacity is non-transferable and implements the following interface to reduce and increase capacity on an MSA.
 
 ```rust
 
-traits Nontransferable {
-  type Balance;
+pub trait Nontransferable {
+    /// Scalar type for representing balance of an account.
+    type Balance: Balance;
 
-  /// The available Capacity for an MSA account.
-  fn balance(msa_id: MessageSourceId) -> Result<Balance, DispatchError>;
+    /// The balance Capacity for an MSA.
```

Comment:
Capacity balance? balance of Capacity?

Reply:
Comments were copied from the code. I've updated both comments to read as the original: 
```rust
    /// The available Capacity for an MSA.
```